backend/fhtways/graph_setup_coord.py

```diff
@@ -56,134 +56,175 @@ def create_graph():
+    G.add_edge("stairs_right_conn", "F4.04_F4.05_c", weight=2, description="Gehen Sie zwei Schritte geradeaus im Korridor. Fühlen Sie die Wand, um sich zu orientieren und die Richtung zu halten.")
+
+
+    G.add_edge("stairs_right_conn", "stairs_right", weight=5, description="Gehen Sie fünf Schritte geradeaus im Korridor. Suchen Sie die Tür auf der rechten Seite am Ende des Korridors. Fühlen Sie nach dem Türgriff, um die Treppe zu erreichen.")
+    G.add_edge("stairs_right", "stairs_right_conn", weight=5, description="Gehen Sie fünf Schritte geradeaus im Korridor. Suchen Sie die Tür auf der rechten Seite am Ende des Korridors. Fühlen Sie nach dem Türgriff, um die Treppe zu erreichen.")
 
-    G.add_edge("stairs_right_conn", "stairs_right", weight=5, description="Gehen Sie den Korridor 5 Schritte geradeaus weiter entlang, bis Sie das Ende des Korridors erreichen. Hier finden Sie eine Tür auf Ihrer rechten Seite. Fühlen Sie nach dem Türgriff, um die Tür zu öffnen und die Stiegen zu erreichen.")
-    G.add_edge("stairs_right", "stairs_right_conn", weight=5, description="Öffnen Sie die Tür vor Ihnen, um in den Korridor der 4. Etage zu gelangen, und biegen Sie unmittelbar links ab. Gehen Sie den Korridor 5 Schritte geradeaus weiter, bis Sie spüren, dass die Wand auf Ihrer linken Seite endet. Dies zeigt das Auftreten eines neuen Korridors auf Ihrer rechten Seite an.")
 
     # Binding the middle part of the hall
-    G.add_edge("door_left", "stairs_middle", weight=8, description="Gehen Sie 5 Schritte geradeaus und biegen Sie nach links ab. Gehen Sie noch 2 Schritte nach vorne und Sie erreichen die Stiegen.")
-    G.add_edge("stairs_middle", "door_left", weight=8, description="Nachdem Sie die 4.Etage erreicht haben, gehen Sie 2 Schritte geradeaus und biegen Sie demnächt nach rechts ab. Gehen Sie geradeaus weiter, bis sie die Türe vor ihnen erreichen, und öffnen Sie diese Tür.")
+    G.add_edge("door_left", "stairs_middle", weight=8, description="Gehen Sie vom linken Eingang zum rechten Eingang. Es sind 14 Schritte geradeaus. Achten Sie auf die Veränderungen im Echo des Raums.")
+    G.add_edge("stairs_middle", "door_left", weight=8, description="Gehen Sie vom linken Eingang zum rechten Eingang. Es sind 14 Schritte geradeaus. Achten Sie auf die Veränderungen im Echo des Raums.")
 
-    G.add_edge("door_left", "door_right", weight=14, description="Door left to Door right.")
-    G.add_edge("door_right", "door_left", weight=14, description="Door right to Door left.")
 
-    G.add_edge("door_left", "lift", weight=10, description="Door left to Lift.")
-    G.add_edge("lift", "door_left", weight=10, description="Lift to Door left.")
+    G.add_edge("door_left", "door_right", weight=14, description="Gehen Sie 14 Schritte geradeaus, um die Tür vor Ihnen zu erreichen. Achten Sie auf die Beschaffenheit des Bodens und die Akustik der Umgebung, um Ihre Bewegung zu orientieren.")
```

Comment:
door_left to door_right (and vice versa) should be more descriptive, because it is an entire hall that the user should go through. maybe add a few words notifying them that they need to transition to another corridor through the hall, it it is going to be a long path straight or smth like this. as far as I remember, there is tactile paving in this hall, you can use that as a reference as well.

Reply:
Possible improvement:

    G.add_edge("door_left", "door_right", weight=14, description="Gehen Sie 14 Schritte geradeaus durch die ganze Halle des 4.Stocks, bis Sie die Tür vor Ihnen erreichen. Achten Sie auf die taktilen Pflastersteine unter Ihren Füßen, die Ihnen den Weg weisen, und die Akustik in der Halle des 4.Stocks, um Ihre Orientierung zu verbessern. Sobald Sie die Tür vor sich erreichen.")
    G.add_edge("door_right", "door_left", weight=14, description="Gehen Sie 14 Schritte geradeaus durch die ganze Halle des 4.Stocks, bis Sie die Tür vor Ihnen erreichen. Achten Sie auf die taktilen Pflastersteine unter Ihren Füßen, die Ihnen den Weg weisen, und die Akustik in der Halle des 4.Stocks, um Ihre Orientierung zu verbessern. Sobald Sie die Tür vor sich erreichen.")

Reply:
Great, but what about the last ". Sobald Sie die Tür vor sich erreichen.". Isn't it redundant, since you already have ", bis Sie die Tür vor Ihnen erreichen." before?